gis/grass8/Portfile

```
@@ -1,323 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           wxWidgets 1.0
-PortGroup           github 1.0
-PortGroup           debug 1.0
-PortGroup           compilers 1.0
+PortGroup           obsolete 1.0
 
-github.setup        OSGeo grass 8.0.1
 name                grass8
-set main_version    [join [lrange [split ${version} "."] 0 1] ""]
+version             8.0.1
 revision            1
```

https://guide.macports.org/chunked/development.practices.html#development.practices.rename-replace-port
> 2. increase the version, revision, or epoch, so that users who have this port installed will get notice in port outdated that they should upgrade it…

(i.e. to the replacement port)

```suggestion
revision            2
```

---

Thanks for the reminder!
(I did have in mind to bump the revision, but apparently I missed it.)